docs/sql-ref-syntax-aux-show-create-table.md

```diff
@@ -37,6 +37,10 @@ SHOW CREATE TABLE table_identifier
 
     **Syntax:** `[ database_name. ] table_name`
 
+* **AS SERDE**
+
+    Just for generating Hive DDL for a Hive SerDe table.
```

Comment:
nit: Maybe `Generates DDL for a Hive SerDe table` to be parallel with `Specifies a table or view name...`?

Reply:
> nit: Maybe `Generates DDL for a Hive SerDe table` to be parallel with `Specifies a table or view name...`?

`Generates DDL for a Hive SerDe table` begin with 4 space, it is same with `Specifies a table or view name...`

Reply:
Sorry for the confusion, I am not talking about spaces. I mean using "Generates" instead of "Just for generating"